app/models/manageiq/providers/ovirt/network_manager.rb

```diff
@@ -33,11 +33,7 @@ def self.description
   end
 
   def self.default_blacklisted_event_names
-    %w(
-      scheduler.run_instance.start
-      scheduler.run_instance.scheduled
-      scheduler.run_instance.end
-    )
+    Settings.ems.ems_ovirt_network.blacklisted_event_names
```

Comment:
If we did something like this, can we avoid creating new methods in the subclasses?

```suggestion
    Settings.ems["ems_#{ems_type}"].blacklisted_event_names
```

Reply:
Yeah we could do that we'd have to copy the settings over into all of the subclasses then I think